doc/source/dev/releasing.rst

```diff
@@ -162,13 +175,22 @@ Upload first the wheels and then the sdist::
 **Github Releases:**
 
 Use GUI on https://github.com/scipy/scipy/releases to create release and
-upload all release artifacts.
+upload all release artifacts. At this stage, it is appropriate to push
+the tag and associate the new release (candidate) with this tag in the GUI.
+For example, ``git push origin v1.2.0rc1``. It is useful to check a previous
```

Comment:
Should that be "upstream" instead of "origin"?

Reply:
I think this is a matter of convention -- I'm not even consistent between different computers I work on & often have my `origin` pointing to the repo propers and the remote `tyler_fork` pointing to my fork, or something like that.

Perhaps this could do with some clarification given the potential for confusion though.

Reply:
Would be good to use `upstream` indeed, and clarify that that is `scipy/scipy`. The standard convention is that `origin` is a contributor's personal fork.